pkg/ingester/metrics.go

```diff
@@ -252,6 +255,12 @@ func newIngesterMetrics(
 			AgeBuckets: 6,
 		}),
 
+		maxLocalSeriesPerUser: promauto.With(r).NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
why "max"? Isn't it just the `localSeriesPerUser`?

Reply:
It's a limit, so it's "max allowed per user". It's also matches the `maxSeriesGauge` for the instance limit above.